packages/cli-tools/bin/streamr-mock-data-generate.ts

```diff
@@ -2,9 +2,13 @@
 import { createFnParseInt } from '../src/common'
 import { createCommand, Options as BaseOptions } from '../src/command'
 import { randomString } from '@streamr/utils'
+import crypto from 'crypto'
 
 interface Options extends BaseOptions {
     rate: number
+    binary: boolean
+    minLength?: number
+    maxLength?: number
```

Comment:
Can these be  `undefined` as there are default values in `.option(...)` definitions?

Reply:
Fixed